client/src/components/EncryptCheckbox.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+const Label = styled.label`
+  margin: 10px 0px 5px 0px;
+`;
+
+const Checkbox = styled.input`
+  margin-left: 10px;
+`;
+
+export default function EncryptCheckbox({ value, onChange, label }) {
+  return (
+    <Label>
+      {label}
+      <Checkbox checked={value} onChange={onChange} type="checkbox" />
```

Comment:
why is here a `value` for checked? since it's a `checkbox` it would make sense to have *checked* as the value..

Reply:
right, thanks)